src/tools/ScenarioMeasurement/Startup/Startup.cs

```diff
@@ -29,7 +29,7 @@ class Startup
         /// <param name="logFileName">optional log file. Default is appExe.startup.log</param>
         /// <param name="workingDir">optional working directory</param>
         /// <param name="warmup">enables/disables warmup iteration</param>
-        /// <param name="traceFileName">trace file name</param>
+        /// <param name="traceName">trace file name</param>
```

Comment:
Did the semantics of this variable change or is this just a cosmetic change?

Reply:
The semantics changed because we can end up having different extensions for Windows and Linux trace files, so having a trace name but not file name here makes more sense. Yeah I should change the its description as well.